public/locale/en.json

```diff
@@ -942,14 +942,14 @@
   "departments_or_teams": "Departments/Teams",
   "describe_why_the_asset_is_not_working": "Describe why the asset is not working",
   "description": "Description",
-  "details_about_the_equipment": "Details about the equipment",
-  "details_of_assigned_facility": "Details of assigned facility",
-  "details_of_origin_facility": "Details of origin facility",
-  "details_of_patient": "Details of patient",
+  "details_about_the_equipment": "Details About The Equipment",
+  "details_of_assigned_facility": "Details of Assigned Facility",
+  "details_of_origin_facility": "Details of Origin Facility",
+  "details_of_patient": "Details of Patient",
   "details_of_shifting_approving_facility": "Details of shifting approving facility",
   "device": "Device",
   "device_associated_successfully": "Device associated successfully",
-  "device_association_exist": "Device association exist!",
+  "device_association_exist": "Device Association Exist!",
```

Comment:
Hmm, I am not sure about this...why adjust this but now the one above? Similar reasoning applies elsewhere in this file as well.